lewis_emulators/knr1050/interfaces/stream_interface.py

```diff
@@ -5,6 +5,7 @@
 from lewis_emulators.utils.command_builder import CmdBuilder
 
 if_connected = conditional_reply('connected')
+if_input_error = conditional_reply('input_correct', "ERROR:20,Instrument in standalone mode")
```

Comment:
See [commit](https://github.com/ISISComputingGroup/EPICS-DeviceEmulator/commit/637f020436dee5c4c3b487bca24144525918279c). Dom and I extended conditional reply a while back so that it can accept a string return. This allows you to test against a reply in the event of an input error etc.

Reply:
:+1: I was mistaken; this is good